Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/AnalyticsController.cs

```diff
@@ -74,7 +75,7 @@ public void Identify(IWeb3Identity? identity)
 
         private void TrackSystemInfo()
         {
-            analytics.Track(AnalyticsEvents.General.SYSTEM_INFO_REPORT, new JsonObject
+            Track(AnalyticsEvents.General.SYSTEM_INFO_REPORT, new JsonObject
```

Comment:
Any reason why we changed from `analytics.Track` to `Track`

Reply:
yep, because Track rise the error message if it was called before `Identify`